src/main/middleware/draftMiddleware.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from 'express'
+import * as moment from 'moment'
+
+import { Draft } from '@hmcts/draft-store-client/dist/draft/draft'
+import { DraftService } from '@hmcts/draft-store-client/dist/draft/draftService'
+
+import { DraftDocument } from '../model/draftDocument'
```

Comment:
Can we not use relative imports?

Reply:
I thought there is not much point since all the imports are more or less at the same depth.

Reply:
The problem I got was that `import { UUIDUtils } from 'utils/uuidUtils'` when compiled becomes `var uuidUtils_1 = require("utils/uuidUtils");` instead of `var uuidUtils_1 = require("../utils/uuidUtils");`.

This is known issue. It looks like `baseUrl` i `paths` are intended more for resource loaders then compilers. Dropped the idea to not spend any more time on it.